pkg/operation/care/checker.go

```diff
@@ -559,6 +555,24 @@ func (b *HealthChecker) CheckControlPlane(
 	return nil, nil
 }
 
+// CheckShootControlPlane checks whether the shoot control plane components in the given listers are complete and healthy.
+func (b *HealthChecker) CheckShootControlPlane(
```

Comment:
Why is there no `CheckGardenControlPlane`? Maybe we should keep it in sync and be more consistent, i.e. either drop `CheckShootControlPlane` or introduce `CheckGardenControlPlane`.